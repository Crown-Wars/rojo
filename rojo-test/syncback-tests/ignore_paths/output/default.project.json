{
  "name": "ignore_paths",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$path": "src"
    }
  },
  "syncbackRules": {
    "ignorePaths": [
      "src/*.rbxm",
      "src/folder_name/**"
    ]
  }
}