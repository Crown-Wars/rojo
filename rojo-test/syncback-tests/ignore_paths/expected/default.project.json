{
  "name": "ignore_paths",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$properties": {
        "CSGAsyncDynamicCollision": {
          "Enum": 0
        },
        "DecreaseMinimumPartDensityMode": {
          "Enum": 0
        },
        "SignalBehavior": "Deferred",
        "StreamOutBehavior": "Opportunistic",
        "StreamingEnabled": true,
        "StreamingIntegrityMode": "PauseOutsideLoadedArea"
      },
      "$path": "src"
    }
  },
  "syncbackRules": {
    "ignorePaths": [
      "src/*.rbxm",
      "src/folder_name/**"
    ]
  }
}